packages/@coorpacademy-components/src/template/back-office/brand-update/style.css

```
@@ -49,35 +49,18 @@
   margin-bottom: 0;
 }
 
-.contentHandler {
-  padding: 10px 40px;
-  display: flex;
-  justify-content: space-between;
-}
-
-.dashboardAside {
+.leftMenu {
   scroll-behavior: unset;
   min-width: 223px;
   background: xtraLightGrey;
 }
 
-.contentView {
-  flex-grow: 1;
-  width: 980px;
-  text-align: left;
-  position: relative;
-  margin: 0 auto;
-  line-height: 1.4em;
-}
-
-.dashboardContent {
-  padding: 1rem;
-  text-align: left;
-  position: relative;
-  margin: 0 auto;
-  margin-top: 16px;
-  line-height: 1.4em;
-  border: 1px solid light;
+.contentWrapper {
+  width: 100%;
+  padding: 0px 64px;
```

```suggestion
  padding: 0 64px;
```

Eslint warns about this